RailwayCo/Assets/Scripts/GameLogic/Currency/RangedCurrencyManager.cs

```diff
@@ -1,10 +1,15 @@
-﻿using System;
+﻿using Newtonsoft.Json;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 
 public class RangedCurrencyManager : ICloneable, IEquatable<RangedCurrencyManager>
 {
-    public Dictionary<CurrencyType, IntAttribute> CurrencyRangedDict { get; private set; }
+    [JsonProperty]
+    private Dictionary<CurrencyType, IntAttribute> CurrencyRangedDict { get; set; }
```

Comment:
so private or public?
And naming convention

Reply:
Huh? It is a private property. I changed it from public to private.

Reply:
Fixed naming convention in 00c00a4